index.html

```diff
@@ -122,16 +137,20 @@ <h2> Tonic</h2>
 
       <div class="Card-Works">
         <div class="Snapshoot-Portfolio">
-          <img src="./media/multi-post.png" alt="Website">
+          <img src="./media/Snapshoot Portfolio-4.png" alt="Website">
         </div>
 
         <div class="Left-Block">
           <div class="Primary-Text">
             <h2> Multi-Post Stories</h2>
 
             <ul>
-              <li>CANOPY</li>
-              <li>Back End Dev</li>
+              <li>FACEBOOK</li>
+              <li class="bullet">               
+              </li>
+              <li>Full Stack Dev</li>
+              <li class="bullet">               
+              </li>
               <li>2015</li>
```

Comment:
- [ ] As requested by the previous reviewer, kindly try to match the Figma design as much as possible, please update the text `color` and the `font-weight` 👍 

| Your design   | Figma design  |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/50721479/199693428-b2367bb8-d8fc-479c-aa1a-9479adad7b3d.png)  | ![image](https://user-images.githubusercontent.com/50721479/199693464-4855e173-1c14-4092-913b-097c38e994ee.png)  |